### ISSUE TITLE
Cannot delete several rows in data editor

### ISSUE THREAD

**Issue by v2e** [veusz/veusz#186]
I created a dataset, entered some values, and later decided to delete a number of consecutive rows --  starting with certain one and to the end of the table. I can select them by pressing Shift and clicking on the first one and the last one. However, it turns out I cannot delete them.

If I right-click and choose "Delete row", only one row is deleted. If I press 'Del' on the keyboard, nothing happens. This is a rather strange behavior, since deleting a bunch of rows is a common task when editing the data tables. Maybe it is a bug?

**Comment by jeremysanders** [comment]
Yes - deleting multiple rows is not yet supported. The data edit dialog could do with some more work.
